src/prefect/flows.py

```diff
@@ -486,8 +486,10 @@ def to_deployment(
         self,
         name: str,
         interval: Optional[Union[int, float, datetime.timedelta]] = None,
+        anchor_date: Optional[Union[str, datetime]] = None,
```

Comment:
hmm i think this might be crowding the interface - what would you think of instead having:
- interval
- cron
- rrule
- schedule (accepts "real" `Schedule` objects for advanced configuration, subsumes timezone and anchor date)

That way it's always forward compatible, and reduces the number of coupled kwargs